plan/plans/show.go

```diff
@@ -128,6 +128,9 @@ func (s *ShowPlan) GetFields() []*field.ResultField {
 			"sql_mode", "Definer", "character_set_client", "collation_connection", "Database Collation"}
 		types = []byte{mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar,
 			mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar}
+	case stmt.ShowProcedureStatus:
+		names = []string{}
```

Comment:
We return empty result. I have tested it with MySQLWorkbench and the error log disappeared.